server/queue/queue-activemq/src/test/java/org/apache/james/queue/activemq/ActiveMQMailQueueBlobTest.java

```diff
@@ -23,47 +23,126 @@
 import java.io.IOException;
 import java.io.InputStream;
 
+import javax.jms.ConnectionFactory;
+
 import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.broker.BrokerPlugin;
+import org.apache.activemq.broker.BrokerService;
+import org.apache.activemq.broker.region.policy.PolicyEntry;
+import org.apache.activemq.broker.region.policy.PolicyMap;
+import org.apache.activemq.plugin.StatisticsBrokerPlugin;
 import org.apache.commons.io.FileUtils;
 import org.apache.james.filesystem.api.FileSystem;
+import org.apache.james.metrics.api.NoopMetricFactory;
+import org.apache.james.queue.api.DelayedMailQueueContract;
+import org.apache.james.queue.api.DelayedManageableMailQueueContract;
+import org.apache.james.queue.api.MailQueue;
+import org.apache.james.queue.api.MailQueueContract;
+import org.apache.james.queue.api.ManageableMailQueue;
+import org.apache.james.queue.api.ManageableMailQueueContract;
+import org.apache.james.queue.api.PriorityMailQueueContract;
+import org.apache.james.queue.api.PriorityManageableMailQueueContract;
+import org.apache.james.queue.api.RawMailQueueItemDecoratorFactory;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableList;
 
-public class ActiveMQMailQueueBlobTest extends ActiveMQMailQueueTest {
+public class ActiveMQMailQueueBlobTest  implements MailQueueContract, ManageableMailQueueContract, DelayedMailQueueContract,
+    DelayedManageableMailQueueContract, PriorityMailQueueContract, PriorityManageableMailQueueContract {
 
-    public static final String BASE_DIR = "file://target/james-test";
-    private MyFileSystem fs;
+    private static final String BASE_DIR = "file://target/james-test";
```

Comment:
you can omit modifiers like private in tests

Reply:
we never did that before, but I agree this could be a good convention for junit 5 tests